websites/S/Sürat Kargo/presence.ts

```diff
@@ -0,0 +1,93 @@
+      presenceData.state = "Sürat Tahsilatlı";
+      presenceData.startTimestamp = Date.now();
+    }else if(ayrac[2] == "SuratKrediliKargo"){
+      presenceData.state = "Sürat Kredili Kargo";
+      presenceData.startTimestamp = Date.now();
+    }else if(ayrac[2] == "SuratBilgilendirme"){
+      presenceData.state = "Sürat Bilgilendirme";
+      presenceData.startTimestamp = Date.now();
+    }else if(ayrac[2] == "SuratAmbalaj"){
+      presenceData.state = "Sürat Ambalaj";
+      presenceData.startTimestamp = Date.now();
+    }
+    // Bilgi Hizmetleri
+  }else if(path.startsWith("/BilgiHizmetleri")){
+    presenceData.details = "Bilgi Hizmetleri:";
+    if(ayrac[2] == "EFaturaSorgulama"){
+      presenceData.state = "E-Fatura Sorgulama";
+      presenceData.startTimestamp = Date.now();
+    }else if(ayrac[2] == "BilgiGuvenligi"){
+      presenceData.state = "Bilgi Güvenliği";
+      presenceData.startTimestamp = Date.now();
+    }else if(ayrac[2] == "BilgiToplumuHizmetleri"){
+      presenceData.state = "Bilgi Toplumu Hizmetleri";
+      presenceData.startTimestamp = Date.now();
+    }else if(ayrac[2] == "BilgiToplumuHizmetleri"){
+      presenceData.state = "Fiyatlandirma";
+      presenceData.startTimestamp = Date.now();
+    }
+      // Kampanyalar
+    }else if(path.startsWith("/Kampanyalar")){
+      presenceData.details = "Kampanyalar";
+      delete presenceData.state;
+      presenceData.startTimestamp = Date.now();
+    }else if(path.startsWith("/BizeUlasin")){
+      presenceData.details = "Bize Ulaşın:";
+      presenceData.state = ayrac[2]
+      presenceData.startTimestamp = Date.now();
+    }else if(path.startsWith("/Basvurular")){
+      presenceData.details = "Başvuru yapıyor:";
+      presenceData.state = ayrac[2]
+      presenceData.startTimestamp = Date.now();
+    }else if(path.startsWith("/BasinKiti")){
+    presenceData.details = "Basın Kiti";
+    delete presenceData.state;
+    presenceData.startTimestamp = Date.now();
+  }else if(path.startsWith("/Haberler")){
+    presenceData.details = "Haber";
+    delete presenceData.state;
+    presenceData.startTimestamp = Date.now();
+  }else if(path.startsWith("/Hakkimizda")){
+    presenceData.details = "Hakkımızda";
+    delete presenceData.state;
+    presenceData.startTimestamp = Date.now();
+  }else if(path.startsWith("/Felsefemiz")){
+    presenceData.details = "Misyonumuz & Vizyonumuz ";
+    delete presenceData.state;
+    presenceData.startTimestamp = Date.now();
+  }else if(path.startsWith("/InsanKaynaklariPolitikamiz")){
+    presenceData.details = "İnsan Kaynakları Politikamız";
+    delete presenceData.state;
+    presenceData.startTimestamp = Date.now();
+  }else if(path.startsWith("/SosyalSorumlulukCalismalarimiz")){
+    presenceData.details = "Sosyal Sorumluluk Çalışmalarımız";
+    delete presenceData.state;
+    presenceData.startTimestamp = Date.now();
+  }else if(path == "/"){
+    presenceData.details = "Ana Sayfa";
+    delete presenceData.state;
```

Comment:
why are you doing delete state when its not even set?

Reply:
fixed